index.rst

```diff
@@ -123,7 +130,7 @@ Here we include a quick analysis to verify that it really was 42 seconds and who
    Maximum duration of tracking: 929.5563025474548
    Minimum duration of tracking: 0.39732980728149414
 
-::
+
 
 These values appear because in each night there are 2 or 3 tracking with a duration of less than 2 seconds and about 5 tracking with a longer duration (some up to 15 minutes).
```

Comment:
Just a question here. And those tracking are still reported to to have "endReason == TMAState.SLEWING" with no faults? All tracks were associated with the block that was running?

Reply:
I have a question about the conclusions. I'm not entirely sure I understand them. Can we say that the rms was never higher than the required value after 3 seconds of tracking?

Reply:
That is, the rms never exceeded the required value after 3 seconds of tracking.